ci/blackbox/rdsbroker_test.go

```diff
@@ -253,106 +275,105 @@ var _ = Describe("RDS Broker Daemon", func() {
 		})
 	})
 
+	var _ = Describe("MySQL Final snapshot enable/disable", func() {
```

Comment:
This indention is accidental. I believe, both `vim-go` and atom with `go-plus` run `go fmt` on save, which triggered the change for us. I'm not sure, how has this been a part of `master` branch before that.